### PR TITLE
Add k8s adminer

### PR DESCRIPTION
**Pull request by YuryHrytsuk**
## What do these changes do?

These changes introduce `traefik` and `adminer` charts which can be installed as follows:
* `helm install adminer ./adminer`
* `helm install trafeik traefik/traefik -f ./charts/traefik/override_values.yaml` # you need helm install traefik/traefik before

The commands above I ran on the k8s cluster machines with copying all `adminer` / `traefik` before .

This will start these 2 services and make `/adminer` and `/ping` endpoints available on port `32080` on the public IP of the machine.

FYI: @matusdrobuliak66 @sanderegg @pcrespov 

## Related issue/s
* https://github.com/ITISFoundation/osparc-ops-environments/issues/669

## Related PR/s
* https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/630

## Checklist
- [ ] I tested and it works

### CODE REVIEW

File: charts/adminer/templates/tests/test-connection.yaml
Comment (mrnicegyu11) [inline charts/adminer/templates/tests/test-connection.yaml:1]: could you please quickly explain this file? thanks

File: charts/adminer/templates/serviceaccount.yaml
Comment (mrnicegyu11) [inline charts/adminer/templates/serviceaccount.yaml:1]: could you please quickly explain this file? thanks

**Review by mrnicegyu11 — APPROVED** [review]
Looks good thanks, excited to learn more about it. I realized that for many file sin the adminer folder I dont get their reason/purpose, did you create them or do they automatically come with adminer?

**Review by mrnicegyu11 — APPROVED** [review]

**Review by mrnicegyu11 — APPROVED** [review]

File: charts/adminer/templates/serviceaccount.yaml
Comment (YuryHrytsuk) [reply charts/adminer/templates/serviceaccount.yaml:1]: discussed in person. These are automatically generated by helm templates

File: charts/adminer/templates/tests/test-connection.yaml
Comment (YuryHrytsuk) [reply charts/adminer/templates/tests/test-connection.yaml:1]: discussed in person. These are automatically generated by helm templates

File: charts/adminer/templates/hpa.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          averageUtilization: {{ .Values.autoscaling.targetCPUUtilizationPercentage }}
+    {{- end }}
+    {{- if .Values.autoscaling.targetMemoryUtilizationPercentage }}
+    - type: Resource
+      resource:
+        name: memory
+        target:
+          type: Utilization
+          averageUtilization: {{ .Values.autoscaling.targetMemoryUtilizationPercentage }}
+    {{- end }}
```
Comment (sanderegg) [inline charts/adminer/templates/hpa.yaml:31]: so this autoscales the adminer?

**Review by sanderegg — COMMENTED** [review]
cool! I guess at some point we should do a workshop...

File: charts/adminer/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: adminer
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
```
[the remark this answers is not in the document]
Comment (YuryHrytsuk) [reply charts/adminer/Chart.yaml:5]: this is generated automatically by `helm`. I don't think it is of importance at this point

File: scripts/run_k8s_locally.bash [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+# For ARM64
+[ "$(uname -m)" = aarch64 ] && curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.23.0/kind-linux-arm64
+chmod +x ./kind
+sudo mv ./kind /usr/local/bin/kind
+
+# install kubectl
+## source: https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl.sha256"
```
[the remark this answers is not in the document]
Comment (YuryHrytsuk) [reply scripts/run_k8s_locally.bash:20]: this script is not robust as of now. Most of the testing was done on remote servers. The local development scripts are to be improved.

having said that, you will need to adjust download link

**Comment by YuryHrytsuk** [comment]
> cool! I guess at some point we should do a workshop...

This is a good idea. It will be also helpful for me to revise / increase further my knowledge of kubernetes

File: charts/adminer/templates/hpa.yaml (hunk 1 above)
Comment (YuryHrytsuk) [reply charts/adminer/templates/hpa.yaml:31]: if it is enabled in values.yaml which as not the case at this moment

**Pull request merged**